wd_core.au3

```diff
@@ -818,7 +818,7 @@ EndFunc   ;==>_WD_ElementAction
 ;                  $bAsync     - [optional] Perform request asyncronously? Default is False
 ;                  $sJSONNode  - [optional] Return the designated JSON node instead of the entire JSON string. Default is ""
 ; Return values .: Success - Raw response from web driver or value requested by given $sJSONNode
-;                  Failure - "" (empty string) and sets @error to one of the following values:
+;                  Failure - "" (empty string) and set @error to $_WD_ERROR_RetValue, or raw response from web driver and sets @error to one of the following values:

```

Comment:
This is getting too complicated in my opinion. Is there a reason that we can't simply return an empty string in all cases? What value comes from returning the non-empty JSON response when there is an error (I haven't checked the code on this myself)?

Also, you changed `sets` to `set`. What that intentional?

Reply:
> Also, you changed `sets` to `set`. What that intentional?

my fault, will change.

As to question about:
`Failure - "" (empty string) and set @error to $_WD_ERROR_RetValue, or raw response from web driver and sets @error to one of the following values:`

this part
`raw response from web driver and sets @error to one of the following values:`

comes from
`$sResponse = __WD_Post($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/execute/" & $sCmd, $sData)`

and
`"" (empty string) and set @error to $_WD_ERROR_RetValue`
comes from
`$sResponse = Json_Get($oJSON, $sJSONNode)`

Reply:
`Also, you changed sets to set. What that intentional?`

I checked and found that 'sets' was provided by this 
https://github.com/Danp2/WebDriver/commit/5267f95a84e4ae07106a8c5104ebc686fadd03d9

Reply:
`Should I change all the .... sets @error to .... set @error ?`
@Danp2  what is your's decision ?

Reply:
I would say no. IMO `sets` is the correct wording in the instance because we are implicitly stating that the function does something. Therefore, "the function sets @error" is correct tense where as "the function set @error" isn't.